.github/workflows/main.yml

```diff
@@ -30,9 +30,9 @@ jobs:
       - name: Production Build
         run: |
           npm run build
-      - name: Unit Tests
-        run: |
-          npm run test
```

Comment:
you need to set the environment variable CI=true

https://create-react-app.dev/docs/running-tests/#continuous-integration